### PR TITLE
129-Fine-tune-berts

### PR DESCRIPTION
**Pull request by GustawB**

### CODE REVIEW

File: src/bert_finetuning/finetuner.py [hunk 1]
```diff
@@ -257,22 +258,22 @@ def train_model(model: callable, dataset: Dataset, labels: list, run_name: str,
...
+                "epochs": 30,
                 "weight_decay": 0.01,
                 "model_name": "bert_multiling_cased",
             }
         )
 
     lr_container = LrContainer(__LEARNING_RATE)
     args = TrainingArguments(
-        "zpp-murmuras/bert",
+        "zpp-murmuras/",
```
[the remark this answers is not in the document]
Comment (GustawB) [reply src/bert_finetuning/finetuner.py:272]: Change is indeed correct

**Comment by SzymonKozl** [comment]
My questions/suggestions before tomorrow (03.04) meeting:
* do we have any metrics on data from edeka+penny datasets? - I did not find the direct information in notebooks
* If we want to use train loss charts in thesis, we need to smoothen them/remove outliers as they are unreadable at this moment (this is not required in this PR anyway)
* why finetune on rev2? - does this affect train/test splits?
* we need to organize large number of models on hf
